web/packages/teleterm/src/ui/TopBar/Identity/IdentitySelector/UserIcon.tsx

```diff
@@ -16,25 +16,43 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import { MouseEvent } from 'react';
 import styled from 'styled-components';
 
-interface UserIconProps {
-  letter: string;
-}
+import {
+  ProfileColor,
+  profileColorMapping,
+} from 'teleterm/ui/services/workspacesService';
 
-export function UserIcon(props: UserIconProps) {
-  return <Circle>{props.letter.toLocaleUpperCase()}</Circle>;
+export function UserIcon(props: {
+  letter: string;
+  /** If not provided, a default neutral color is rendered. */
+  color?: ProfileColor;
+  className?: string;
```

Comment:
`className` is unused?

Reply:
Passing `className` to the component (and then down to some element) allows it to be wrapped in `styled()` or have a `css` prop applied. I used it add some custom styling, but actually it's better to add more props to `UserIcon` and keep everything there. I refactored it and got rid of `className`.